libraries/AC_AttitudeControl/AC_PosControl.h

```diff
@@ -450,6 +456,8 @@ class AC_PosControl
     float       _jerk_max_xy_cmsss;       // Jerk limit of the xy kinematic path generation in cm/s^3 used to determine how quickly the aircraft varies the acceleration target
     float       _jerk_max_z_cmsss;        // Jerk limit of the z kinematic path generation in cm/s^3 used to determine how quickly the aircraft varies the acceleration target
     float       _vel_z_control_ratio = 2.0f;    // confidence that we have control in the vertical axis
+    Vector2f    _disturb_pos;
```

Comment:
it would be nice to have comments besides these variables like the ones above it.

Reply:
done.  Thanks!